docs/api-reference/io-columns-regression.md

```diff
@@ -0,0 +1,6 @@
+### Input and Output Columns
+The input label column data must be <xref:System.Single>. This trainer outputs the following columns:
+
+| Output Column Name | Column Type | Description|
+| -- | -- | -- |
+| `Score` | <xref:System.Single> | The unbounded score that was calculated by the trainer to determine the prediction.||
```

Comment:
Is "unbounded score" clear enough here? It is the raw result of the regression calculation right?